packages/react-kit/src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { FooterProps } from './props';
+
+/**
+ * @see {@link https://live-storybook-react-kit.appa.pantheon.site/?path=/docs/footer-footer--docs}
+ */
+export const Footer = ({ Logo, Link }: FooterProps) => {
+	const FooterLogo = () => {
+		if (React.isValidElement(Logo)) {
+			return Logo;
+		} else if (typeof Logo === 'object' && 'src' in Logo && 'alt' in Logo) {
+			const { src, alt } = Logo;
+			return <img src={src} height="40" alt={alt} className="rk-inline" />;
+		}
+		return null;
+	};
+	return (
+		<footer className="rk-mt-16 rk-bg-neutral-800 rk-py-4 rk-text-center rk-text-white">
```

Comment:
There should be no top margin here

Reply:
Hmm true but should we implement it on the footer itself or on the page/pages that the footer resides?